shared/src/helpers/debounce/use-debounce/use-debounce.helper.ts

```diff
@@ -0,0 +1,8 @@
+import { debounce } from 'debounce';
+
+const useDebounce = (
```

Comment:
It's not a hook, so let's rename it to something like this. Also don't forget to rename file and folder
```suggestion
import { debounce as debounceCallback } from 'debounce';

const debounce = (
```